llvm/test/CodeGen/AMDGPU/required-export-priority.mir

```diff
@@ -0,0 +1,293 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn -mcpu=gfx1150 -mattr=-wavefrontsize32,+wavefrontsize64 -run-pass=post-RA-hazard-rec -verify-machineinstrs  %s -o - | FileCheck -check-prefixes=GFX1150 %s
```

Comment:
Don't need -wavefrontsize32 or -verify-machineinstrs. Also test the generic target?

Reply:
Done